client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEmptyForm.jsx

```diff
@@ -74,7 +74,7 @@ const SubmissionEmptyForm = (props) => {
       <Button
         color="primary"
         disabled={isSaving}
-        onClick={handleSaveGrade}
+        onClick={() => handleSaveAllGrade()}
```

Comment:
You can pass `handleSaveAllGrades` directly since it is already a function without any argument. No need to recreate another function for this. (Explain why)